Course (#152)
whitemech/whitemech.github.io: _pages/courses.md

---
title: "WhiteMech - Courses"
layout: textlay
excerpt: "WhiteMech -- Courses"
sitemap: false
permalink: /courses
---

# Game-Theoretic Approach to Planning and Synthesis

- **Instructors**: [Giuseppe De Giacomo](https://www.dis.uniroma1.it/degiacom/), [Antonio Di Stasio](https://www.dis.uniroma1.it/users/antonio_di-stasio), [Giuseppe Perelli](https://giuseppeperelli.github.io/), [Shufang Zhu](https://shufang-zhu.github.io/) 

> - **Type**: PhD Course.

Reviewer: ```suggestion
- **Type**: PhD Course
```

> - **Number of hours**: 20.

Reviewer: ```suggestion
- **Number of hours**: 20
```

- **Class delivery mode**: Blended

> - **Dates**: From Monday 4th July to Friday 8th July from 14:00 to 18:00

Reviewer: Maybe you want to add that it's on CEST time zone?

> - **In presence location**: Room 7, Department of Computer, Automation, and Business Engineering, Sapienza University of Rome

Reviewer: ```suggestion
- **In presence location**: Room A7, Department of Computer, Automation, and Management Engineering, Sapienza University of Rome
```
